src/features/pages/partners/deriv-prime/why-deriv-prime/why-deriv-prime.module.scss

```diff
@@ -0,0 +1,29 @@
+@use 'features/styles/theme/theme-mixins' as *;
+@import 'features/styles/theme/variables';
+
+.container {
+    inline-size: auto ;
+
+    @include breakpoints(laptop) {
+        inline-size: 124rem;
+    }
+}
+
+.header {
+   margin-inline: 20%;
+}
+
+.card {
+    box-shadow: 0px 12px 16px -4px rgba(14, 14, 14, 0.08), 0px 4px 6px -2px rgba(14, 14, 14, 0.03);
+    inline-size: 95%;
+    block-size: auto;
+
+    @include breakpoints(phone) {
+        inline-size: 38.9rem;
+    }
```

Comment:
Can we use flex-basis classes such as `flex-basis-1-3` instead of the width?

Reply:
With flex-basis only the cards will look different because some of them have more text than the others. Will need to define inline-size in order to have consistency